tidy3d/plugins/dispersion/fit.py

```diff
@@ -697,3 +697,63 @@ def from_file(cls, fname: str, **loadtxt_kwargs):
         else:
             wvl_um, n_data, k_data = data.T
         return cls(wvl_um=wvl_um, n_data=n_data, k_data=k_data)
+
+    @classmethod
+    def from_complex_permittivity(
+        cls,
+        wvl_um: ArrayFloat1D,
+        eps_real: ArrayFloat1D,
+        eps_imag: ArrayFloat1D = None,
+        wvl_range: Tuple[Optional[float], Optional[float]] = (None, None),
+    ):
+        """Loads :class:`DispersionFitter` from wavelength and complex permittivity data
```

Comment:
perhaps we should mention here or in the parameters that it's `relative` permittivity? (and for the one below)